doc/source/overview.rst

```diff
@@ -11,24 +11,70 @@ Here are the 5 levels of sharing your code. We provide tutorials for each level.
 
 .. include:: snippets/5-levels-table.rst
 
-⏩️ Are you here to **start a new Python** project?
+Where do I begin?
+-----------------
 
-    If you are **new** to programming, start from Level 1. You will learn how to reuse code across files and folders. You will also learn how to write unit tests. To get started, visit :ref:`level-1-2-3-tutorials`.
+I am here to **start a new Python** project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    If you have **experience** developing scientific code in Python, we recommend starting from Level 4, where you will create a lightweight Python package with automatic formatting and unit testing using GitHub Actions. To get started, visit :ref:`level-4-tutorial`.
+    ⏩️ If you have **experience** developing scientific code in Python, we recommend starting from Level 4, where you will create a lightweight Python package with automatic formatting and unit testing using GitHub Actions.
 
-    If you are an **active open-source developer** and you are also familiar with GitHub Actions and forking workflows, we recommend you start from Level 5. To get started, visit :ref:`level-5-tutorial`.
+        .. code-block:: bash
 
-⏩️ Are you here to **migrate** an existing Python project?
+            conda create -n skpkg-env scikit-package
+            conda activate skpkg-env
+            package create system
 
-    Do you want to migrate your existing Python project/package to the ``scikit-package`` Level 5 ``public`` standard? To get started, visit :ref:`migrate-existing-package-to-level-5`.
```

Comment:
I found that instead of "Do you want to migrate your existing Python.. to "I want to migrate my existing Python..." reads smoothly since the user don't have to convert "you" to "I" vice versa.